### PR TITLE
Add xeus recipe

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Using cmake as per #2168 .

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Yay! Success...

File: recipes/xeus/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/xeus/bld.bat]: Is there a `ctest` we can run?

File: recipes/xeus/build.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR
+make install
```
Comment (ocefpaf) [inline recipes/xeus/build.sh]: `make check`?

File: recipes/xeus/bld.bat (hunk 1 above)
Comment (SylvainCorlay) [inline recipes/xeus/bld.bat]: We have an example echo kernel that builds and runs. I will add this to the feedstock. I wanted to iterate on this.

**Pull request merged**